circle.yml

```diff
@@ -34,3 +34,11 @@ deployment:
           docker push ${DOCKER_IMAGE}:${CIRCLE_TAG}
         fi
 
+        # On master merges, ensure we tag the new version.
+        [[ "${CIRCLE_BRANCH}" == "master" ]] || exit 0
```

Comment:
Is there a reason you don't want to run this on other branches? It'll generate a nice tag like v2.3.0-<sha>, which allows one to pull/run builds from branches.

Reply:
That's a good point. I'll remove the `master` only check.